src/main/java/www/purple/mixxy/models/ComicsDto.java

```diff
@@ -0,0 +1,14 @@
+package www.purple.mixxy.models;
+
+/**
+ * Created by Brian_Sabz on 4/7/16.
+ * 
+ * @author Brian_Sabz
+ */
+import java.util.List;
+
+public class ComicsDto {
```

Comment:
You know, I'm not sure we need this TBH.  A bit of brain-picking on IRC leaves me thinking that DTOs are more useful for *submitting* data than for receiving it, at least in our case.  When submitting, we can just fill out the raw data like text or images and let the controller, constructor, or even DAO itself figure out the rest.  Maybe for querying DTOs could be useful for narrow views of data (so we don't have fields with `null` or otherwise undefined values).  I'm not quite sold on a DTO with just one field, though.  Is there some other data you can think of that we might associate with a *set* of Comics?